.github/workflows/gpu-ci.yml

```diff
@@ -1,12 +1,29 @@
-# Sets env vars based on GPU platform
 # Prerequisites
-# - Self-hosted Nvidia GPU runner with `gpu-ci` tag in caller repo
+# - Self-hosted Nvidia GPU runner with CUDA enabled
+# - Runner attached in caller repo with `gpu-ci` label
 # - `cuda` and `opencl` Cargo features
-# - `nextest.toml` `ci` profile and `Cargo.toml` `dev-ci` profile
 name: GPU CI Tests 
 
 on:
+  # We expect dependents to call this on `merge_group` and conditionally run with
+  # if: github.event_name != 'pull_request' || github.event.action == 'enqueued' # 
   workflow_call:
+    inputs:
+      # Nextest profile specified in `.config/nextest.toml`
+      nextest-profile:
+        required: false
+        default: "default"
+        type: string
+      # Cargo profile specified in `Cargo.toml`
+      cargo-profile:
+        required: false
+        default: "release"
+        type: string
```

Comment:
I'm not against this in principle, but I think we've kept it uniformly to one choice across our repos. Did you notice otherwise?

Reply:
For the record, I'm ok with either way of managing this. But:
- https://github.com/lurk-lab/neptune/pull/262
- https://github.com/lurk-lab/grumpkin-msm/pull/12